src/snap/entropy.ts

```diff
@@ -1,48 +1,50 @@
 // Copyright (c) Silence Laboratories Pte. Ltd.
 // This software is licensed under the Silence Laboratories License Agreement.
 
-import { SnapError, SnapErrorCode } from '../error';
 import _sodium from 'libsodium-wrappers';
+import { SnapError, SnapErrorCode } from './error';
 
-export const requestEntropy = async (salt?: Uint8Array) => {
-	const hexSalt = salt ?? crypto.getRandomValues(new Uint8Array(32));
-	const entropy = (await snap.request({
-		method: 'snap_getEntropy',
-		params: {
-			version: 1,
-			salt: hexSalt.toString(),
-		},
-	})) as string;
+export default class Entropy {
+	public static requestEntropy = async (salt?: Uint8Array) => {
```

Comment:
Static class only is a bad practice: https://biomejs.dev/linter/rules/no-static-only-class/

We should keep it as a JS module.